samples/data-sealing/common/dispatcher.cpp

```diff
@@ -161,6 +164,7 @@ int ecall_dispatcher::unseal_data(
     size_t seal_key_size = 0;
     uint8_t* key_info = NULL;
     size_t key_info_size = 0;
+    (void)sealed_data_size;
```

Comment:
I'll create an issue to resolve this. Fixing unused parameters in samples we can definitely push off into an issue unrelated to NuGet package support :)

As for the casts that were introduced that truncate possibly larger values (due to what I assume is MSVC compiler warnings/errors), that needs to get resolved before this merges because we're changing behavior.

Reply:
Added issue addressing this: #2220